porter/main.py

```diff
@@ -53,10 +54,14 @@ class Porter(Learner):
     _LONG_LEARNING_DELAY = 30
     _ROUNDS_WITHOUT_NODES_AFTER_WHICH_TO_SLOW_DOWN = 25
 
-    DEFAULT_EXECUTION_TIMEOUT = 15  # 15s
-
     DEFAULT_PORT = 9155
 
+    MAX_GET_URSULAS_TIMEOUT = os.getenv("PORTER_GET_URSULAS_TIMEOUT", default=15)
+    MAX_DECRYPTION_TIMEOUT = os.getenv(
+        "PORTER_MAX_DECRYPTION_TIMEOUT",
```

Comment:
Can envvars be documented somewhere?

Reply:
👍 I was thinking the threshold docs, and the repos readme.

Those will be done separately.